_csharp-api/inputsystem/input.md

```diff
@@ -2,8 +2,82 @@
 
 This static type provides you with input functionality, the base input system has no events to bind to and supports polling only. It's largely self explanatory with a few possible gotchas.
 
-## Key Polling
+## Key Reading
 
-`GetKeyDown(KeyCode)` | `GetKeyPressed(KeyCode)` | `GetKeyReleased(KeyCode)` - These are likely to be the most commonly used methods, `GetKeyDown` will return true on any number of frames so long as a key is down, this can cause undesired behaviour. If you only want to read an input once, use `GetKeyPressed` or `GetKeyReleased`.
+There's a number of usefull methods that detect keys being pressed down in different ways, all seeming to return a 'bool'. Most common are `isKeyDown`, `isKeyPressed` and `isKeyReleased`
```

Comment:
Still not capitalized :(
In C# methods names starts with Upper letter.

Reply:
I know, when you first mentioned it I capitalized the extensions, but totally didn't see the above, more obvious ones. Weirdly enough I did use the correct from everywhere else.